signing_clients/apps.py

```diff
@@ -109,21 +128,18 @@ def _digest(data):
 class Section(object):
     __slots__ = ('name', 'algos', 'digests')
 
-    def __init__(self, name, algos=('md5', 'sha1'), digests={}):
+    def __init__(self, name, digests={}):
```

Comment:
I know this isn't your code but I just noticed it! dictionaries and lists - mutable objects - should **never** be an argument. While you're at it, mind using `None` as a default for this and set it to an empty dictionary in the constructor in case `digests` is `None`? We should check if we have these defaults somewhere else, ugh!